test/libsolidity/util/TestFunctionCall.cpp

```diff
@@ -247,10 +246,23 @@ string TestFunctionCall::formatBytesParameters(
 		}
 		else
 		{
-			ParameterList defaultParameters = ContractABIUtils::defaultParameters((_bytes.size() + 31) / 32);
+			if (call().kind == FunctionCall::Kind::Builtin && call().builtinReturnType)
+			{
+				if (call().builtinReturnType == TypeProvider::boolean())
+				{
+					if (util::toHex(_bytes) == "0000000000000000000000000000000000000000000000000000000000000000")
```

Comment:
Initially I thought that e.g. `BoolType::toString(bool)` returns the textual of the supplied value, but this function returns the type as string, where the bool is `_short` maybe refers to a long vs. a short typename. 
However, maybe we could add something like this to these type, e.g. `string BoolType::valueAsString(bool _value)`.